css/css-multicol/column-count-used-001-ref.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<title>column-count used value test ref</title>
+<style>
+ul {
+    font: 20px/1 Ahem;
```

Comment:
To simplify the test, we could just use DIVs with a blank line created by BR elements. We don't need Ahem, and also not UL or LI. If you set orphans and widows to 1, the contents of the multicol container could simply be
```
<br><br><br><br>
```
(i.e. no need for any LI/DIV wrapper for each line)

It would be even better with a test that would just match one of the refs in ../reference/ , of course ("green square" something).

Reply:
@mstensho does this look better? oddly, it fails in Safari, as the green box is 108px high, and I don't know if I'm missing something as to why.